src/Components/Press_Inquiries_Controller.php

```diff
@@ -7,19 +7,36 @@
 class Press_Inquiries_Controller {
 
 	protected array $block;
+	protected string $panel_id;
 
 	public function __construct( $block ) {
-		$this->block = (array) $block;
+		$this->block    = (array) $block;
+		$this->panel_id = wp_unique_id( 'press-inquiries-block-' );
```

Comment:
if I am not mistaken $block has unique ID already

Reply:
Ah, yeah. I'll switch to just using the block ID.